numpy/__init__.pxd

```diff
@@ -461,6 +469,7 @@ cdef extern from "numpy/arrayobject.h":
     object PyArray_ZEROS(int nd, npy_intp* dims, int type, int fortran)
     object PyArray_EMPTY(int nd, npy_intp* dims, int type, int fortran)
     void PyArray_FILLWBYTE(ndarray, int val)
+    npy_intp PyArray_REFCOUNT(object)
```

Comment:
```suggestion
```
This is the wrong way around, `PyArray_REFCOUNT` should be deleted in the other file, not added here.

Reply:
Not that it matters much, but could you also remove it from the other file?

Reply:
Done